flink-runtime/src/test/java/org/apache/flink/runtime/taskexecutor/slot/TaskSlotTableImplTest.java

```diff
@@ -115,11 +115,11 @@ public void testRetrievingAllActiveSlots() throws Exception {
     }
 
     /**
-     * Tests that redundant slot allocation with the same AllocationID to a different slot is
-     * rejected.
+     * Tests that inconsistent static slot allocation with the same AllocationID to a different slot
+     * is rejected.
      */
     @Test
-    public void testRedundantSlotAllocation() throws Exception {
+    public void testInconsistentStaticSlotAllocation() throws Exception {
```

Comment:
This covers the case trying to allocate the same allocation id to different slot index.
I think we need to also cover the case that different allocation ids trying to take the same slot index